bentoml/types.py

```diff
@@ -6,13 +6,14 @@
 
 # http://www.apache.org/licenses/LICENSE-2.0
 
+import functools
```

Comment:
?

Reply:
My IDE moved the import 2 lines up, I'm not sure why